lib/r10k/cli/puppetfile.rb

```diff
@@ -46,7 +46,7 @@ def self.command
           usage 'check'
           summary 'Try and load the Puppetfile to verify the syntax is correct.'
 
-          required nil, :puppetfile, 'Path to Puppetfile (can also be set with PUPPETFILE environment variable)'
```

Comment:
Gah. I had some back and forth with original branches on the deprecation. In the original commit for RK-161, I put a big "DEPRECATED" inside the parens on the help page. Which do you think is a better choice?